### ISSUE TITLE
How you label the face images in Vggface2 dataset?

### ISSUE THREAD

**Issue by Huixxi** [cydonia999/VGGFace2-pytorch#6]
In the paper, I saw they used two pretrained classification model to get the information of head pose and age to label the images in Vggface2, now how you get those label info? Have you used the same way? And could you please release these pre-trained models?

**Comment by imranparuk** [comment]
The author of this project did a great job, however I don't suppose he/she will be supporting it. (idk maybe lets see)... However I am in the process of re-creating it. Show some love to my project -> [VGGFace2](https://github.com/imranparuk/VGGFace2)

1) Hmm, I didn't notice that one in the paper, but ill give it a better read to understand that.
2) Probably not, and I don't expect mine to be either... but we can aim to get it as close as possible?
3) Pre-trained models are available here if you wish -> [Pre-trained weights?](https://github.com/ox-vgg/vgg_face2) ...However, I have not yet tried importing them as yet.
